test/check.js

```diff
@@ -7,5 +7,5 @@ var postcss = require('postcss');
 module.exports = function check( input, output, opts ) {
   var processor = postcss([lost(opts)]);
 
-  expect(processor.process(input).css).to.equal(output);
```

Comment:
the `expected` is what the result should be (the right result)
the `actual` is what your script is responding